iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothub/TransportClientTests.java

```diff
@@ -50,8 +47,8 @@
     private static final int NUM_MESSAGES_PER_CONNECTION = 3;
 
     private static final long RETRY_MILLISECONDS = 100; //.1 seconds
-    private static final long SEND_TIMEOUT_MILLISECONDS = 5 * 60 * 1000; // 5 minutes
-    private static final long RECEIVE_MESSAGE_TIMEOUT = 5 * 60 * 1000; // 5 minutes
+    private static final long SEND_TIMEOUT_MILLISECONDS = 20 * 1000; // 20 seconds
```

Comment:
Nothing like TimeSpan in Java?

Reply:
Not that I'm familiar with, no. Operations like .await are always just in milliseconds (long or int)